internal/service/project/data_source_project.go

```diff
@@ -166,6 +167,11 @@ func (d *projectDS) Schema(ctx context.Context, req datasource.SchemaRequest, re
 					},
 				},
 			},
+			"tags": schema.MapAttribute{
+				ElementType: types.StringType,
+				Optional:    true,
```

Comment:
In data source fields should be only Computed

Reply:
fixed in 35d729e4e186219cab55b274cb1af334b82e1645